scripts/clear-signed-agreements.py

```diff
@@ -52,22 +50,29 @@ def main(stage, framework_slug, api_token, user, supplier_ids=None):
     suppliers = api_client.find_framework_suppliers(framework_slug, agreement_returned=True)['supplierFrameworks']
```

Comment:
This line is fine, but line 48 keeps breaking for me.

```
# 48
supplier_ids = [int(supplier_id.strip()) for supplier_id in supplier_ids.split(',')]
```

It looks like it's looking for a comma-separated string of supplier ids, which isn't the format ids are stored as in the supplier_id file. 

On line 95, the supplier_id file is looking for a file with supplier ids separated by newlines.  This is also the format we use for bulk document uploading, so I think stick with that.